newIDE/app/src/UI/LocalFolderPicker/index.js

```diff
@@ -8,9 +8,14 @@ import * as React from 'react';
 import TextField from '../TextField';
 import optionalRequire from '../../Utils/OptionalRequire';
 import FlatButton from '../FlatButton';
+import IconButton from '../IconButton';
+import RotateLeftIcon from '@material-ui/icons/RotateLeft';
```

Comment:
Could you use the `CustomSvgIcons/Undo` one? Not the best but it will be coherent with the variable list editor that uses it as a "Restore instance variable value to object variable value"